packages/@vuepress/theme-default/src/client/components/ToggleDarkButton.vue

```diff
@@ -44,17 +44,39 @@ const toggleDark = (): void => {
   isDark.value = !isDark.value
 }
 
-const setDarkClass = (): void => {
+const updateDarkClass = (): void => {
   const htmlEl = window?.document.querySelector('html')
   htmlEl?.classList.toggle('dark', isDark.value)
 }
 
+const updateThemeColor = (): void => {
+  const themeColorEl = window?.document.querySelectorAll(
+    'meta[name="theme-color"]'
+  )
+  if (themeColorEl) {
+    const newThemeColor = window
+      .getComputedStyle(window.document.documentElement)
+      ?.getPropertyValue(isDark.value ? '--c-bg' : '--c-brand')
+      .trim()
```

Comment:
- The question mark in `?.getPropertyValue` seems useless.
- Is the ending `trim()` necessary?

Reply:
1. I think in some case `getComputedStyle()` may return an empty result, so I added a question mark. It did superfluous because if an empty result has been returned, the website must be broken.

2. Without the `trim()`, `newThemeColor` would always got 1 more space the front of color hex, though the space has effects to the results, but seeing that change `content="#xxxxxx"` -> `content=" #xxxxxx"` would let me feel uncomfortable.

I’m sorry that I'm not familiar with the actual principles for the code style. If u still think it is unnecessary, I can change them later.

Reply:
1. Check the MDN docs: https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle
  `getComputedStyle` will always return a live CSSStyleDeclaration object.

2. I didn't see any extra space in chrome. Does it only happen in Safari?

Reply:
1. Thanks, I removed the question mark in commit https://github.com/vuepress/vuepress-next/pull/218/commits/42b3ce7c994d06fe174887ec6ab86f6fe5fd09df

2. In Safari Version 15.0 (17612.1.15.1.14), it would always have an extra space on development (`docs:dev`) but not happen on release. 

<img width="1440" alt="Screen Shot 2021-06-15 at 21 49 46" src="https://user-images.githubusercontent.com/15633984/122066599-77907600-ce25-11eb-8967-6158e96ccef8.png">

Reply:
Got it. It only happens in dev because the CSS is not minified. Let's keep the `trim()` for now.